scripts/localization.swift

```diff
@@ -0,0 +1,349 @@
+import Foundation
```

Comment:
Script for conversion between twn and ios formats. Trying out swift as a scripting language.